VMBackup/main/handle.py

```diff
@@ -72,7 +72,7 @@ def main():
         total_used_size = 0
         size_calculation_failed = False
         eventlogger = None
-        HandlerUtil.waagent.LoggerInit('/dev/console','/dev/stdout')
+        HandlerUtil.waagent.LoggerInit(None,'/dev/stdout')
```

Comment:
it might be better to add list of loggers to initialize in a config and pass those instead of checking specific config in backuplogger?